requirements.txt

```diff
@@ -16,4 +16,6 @@ MAVProxy==1.8.5
 monotonic==1.5
 pyserial==3.4
 urllib3==1.24.2
-opencv-python
\ No newline at end of file
+opencv-python==4.1.1
+socket
```

Comment:
I think socket is a default package that does not need to installed seperately python 3.